src/content/docs/bot/usage.md

```diff
@@ -20,12 +20,13 @@ Comment on Issue or Pull Request, asking @all-contributors to add a contributor:
 
 Also, you can now add multiple contributors at the same time:
 
-##### Using single line
+#### Using single line
```

Comment:
There was an unexpected jump in headers - h3 -->h5 which was making. markdownlint fail. fixed that.